src/Razor/benchmarks/Microsoft.AspNetCore.Razor.Performance/ProjectSystem/FullProjectSnapshotHandleSerializationBenchmark.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using BenchmarkDotNet.Attributes;
+using Microsoft.AspNetCore.Razor.Language;
+using Microsoft.CodeAnalysis.Razor;
+using Microsoft.CodeAnalysis.Razor.ProjectSystem;
+using Microsoft.CodeAnalysis.Razor.Workspaces.Serialization;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.AspNetCore.Razor.Performance
+{
+    public class FullProjectSnapshotHandleSerializationBenchmark
+    {
+        private static readonly string ExpectedFilePath = "C:\\Users\\admin\\location\\blazorserver\\blazorserver.csproj";
```

Comment:
I suspect that this isn't actually machine-dependent, it's just an artifact of the way the test works? If so should add a comment explaining why it works that way.

Reply:
Oh yeah, it's just the hardcoded path in the `project.razor.json` we use for the benchmark: 
![image](https://user-images.githubusercontent.com/14852843/87836597-6e79dc00-c845-11ea-9b5c-8a4053b6150a.png)